gluster-block-basic.yml

```diff
@@ -0,0 +1,65 @@
+- job:
+    name: gluster-block-basic
+    node: gluster
+    description: Build gluster-block
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        branches:
+        - '*/master'
+        git-tool: Default
+        url: https://github.com/gluster/gluster-block.git
+    - git:
+        url: https://github.com/gluster/centosci.git
+        branches:
+        - origin/master
+
+    builders:
+    - shell: !include-raw: scripts/common/get-node.sh
+    # the heketi/ci-tests repo contains a copy of bootstrap.sh from the
+    # gluster/centosci repo
+    - shell: ./scripts/gluster-block/gluster-block-basic.sh "ghprbPullId=${ghprbPullId}"
```

Comment:
You need the bootstrap script here.

Reply:
ah! ok.. heketi one had it, i didn't know the reason, so removed it, will change. Will the argument to that script be fine?

Reply:
Done